sound/soc/sof/control.c

```diff
@@ -432,6 +432,8 @@ int snd_sof_bytes_ext_get(struct snd_kcontrol *kcontrol,
 	 * Decrement the limit by ext bytes header size to
 	 * ensure the user space buffer is not exceeded.
 	 */
+	if (size < sizeof(const struct snd_ctl_tlv))
+		return -EINVAL;
 	size -= sizeof(const struct snd_ctl_tlv);
```

Comment:
Agree with @lyakh , const in sizeof looks strange. We've had enough churn for this code, so let's fix this as well.